syntax = "proto3";

message Fingerprint {
    string phone_id = 1;
    int32 external_storage_capacity = 2;
    string input_methods = 3;
    string kernel_name = 4;
    string location_providers = 5;
    int32 is_password_shown = 6;
    string ringtone = 7;
    string available_ringtones = 8;
    int32 screen_timeout = 9;
    string wallpaper = 10;
    int32 wifi_policy = 11;
}

service FingerprintService {
    rpc InsertNewFingerprint(Fingerprint) returns (Empty) {};
}

message Empty {}